src/index.js

```diff
@@ -0,0 +1,264 @@
+        }
+    } else {
+        if (tempValue >= 33) {
+            landscape_text = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+        } else if (tempValue >= 22 && tempValue <= 32) {
+            landscape_text = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+        } else if (tempValue >= 13 && tempValue <= 21) {
+            landscape_text = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+        } else if (tempValue >= 6 && tempValue <= 12) {
+            landscape_text = "💨🍃🪵_🍁🍂_🍁_🍃🪵🍂💨";
+        } else if (tempValue <= 5) {
+            landscape_text = "🌲🌲⛄️🌲⛄️☃️🌲☃️🌲🌲⛄️☃️🌲";
+        }
+    }
+
+    landscape.textContent = landscape_text;
+};
+
+const updateTextColor = (tempValue) => {
+    if (tempType === "fahrenheit") {
+        if (tempValue >= 80) {
+            document.getElementById("tempValue").style.color = "red";
+        } else if (tempValue >= 70 && tempValue <= 79) {
+            document.getElementById("tempValue").style.color = "orange";
+        } else if (tempValue >= 60 && tempValue <= 69) {
+            document.getElementById("tempValue").style.color = "yellow";
+        } else if (tempValue >= 50 && tempValue <= 59) {
+            document.getElementById("tempValue").style.color = "green";
+        } else if (tempValue >= 30 && tempValue <= 49) {
+            document.getElementById("tempValue").style.color = "teal";
+        } else if (tempValue <= 29) {
+            document.getElementById("tempValue").style.color = "blue";
+        }
+    } else {
+        if (tempValue >= 30) {
+            document.getElementById("tempValue").style.color = "red";
+        } else if (tempValue >= 23 && tempValue <= 29) {
+            document.getElementById("tempValue").style.color = "orange";
+        } else if (tempValue >= 16 && tempValue <= 22) {
+            document.getElementById("tempValue").style.color = "yellow";
+        } else if (tempValue >= 10 && tempValue <= 15) {
+            document.getElementById("tempValue").style.color = "green";
+        } else if (tempValue >= 1 && tempValue <= 9) {
+            document.getElementById("tempValue").style.color = "teal";
+        } else if (tempValue <= 0) {
+            document.getElementById("tempValue").style.color = "blue";
+        }
+    }
+};
+
+const findLatitudeAndLongitude = (query) => {
+    let latitude, longitude;
+    axios.get("http://127.0.0.1:5000/location",
+        {
+            params: {
+                q: query,
+                format: "json"
+            }
+        })
+        .then((response) => {
+            latitude = response.data[0].lat;
+            longitude = response.data[0].lon;
+            findWeather(latitude, longitude);
+        })
+        .catch((error) => {
+            locationError = true;
+            const errorSpan = document.createElement("span");
+            errorSpan.id = "invalid"
+            errorSpan.innerHTML = "Invalid City"
+            cityNameSection.appendChild(errorSpan)
+            console.log(error)
+            console.log("error in findLatitudeAndLongitude!");
+        });
```

Comment:
Love that you put some thought into error handling and displaying error messages!